Task3-1.playground/Contents.swift

```diff
@@ -0,0 +1,157 @@
+/**
```

Comment:
```suggestion
/*
```

please, look at documentation about multiline comments. https://docs.swift.org/swift-book/LanguageGuide/TheBasics.html

Reply:
Thank you for the information.
I used code documentation with `/** */`
[Info here.](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/ComentBlock.html#//apple_ref/doc/uid/TP40016497-CH103-SW1)